docs/Getting Started/Understanding Permissions/index.md

```diff
@@ -94,7 +94,10 @@ function onHmiStatusListener (onHmiStatus) {
         // now in HMI FULL
     }
 }
-sdlManager.addRpcListener(SDL.rpc.enums.FunctionID.OnHMIStatus, onHmiStatusListener);
+
```

Comment:
This is using the new API in the lifecycle config but the paragraph above it still says to use the SdlManager addRpcListener which doesn't make sense with this code snippet.